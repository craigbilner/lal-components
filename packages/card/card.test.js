import 'react-native';
import React from 'react';
import Card from './card';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <Card title="Some sort of property"
          uri="https://lid.zoocdn.com/645/430/14b6a430b76a235e8de2ba02d695e7a0bd789b40.jpg"
    />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
