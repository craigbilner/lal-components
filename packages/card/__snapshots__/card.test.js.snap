// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flexGrow": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 25,
        "fontWeight": "bold",
      }
    }
  >
    Some sort of property
  </Text>
  <Image
    source={
      Object {
        "uri": "https://lid.zoocdn.com/645/430/14b6a430b76a235e8de2ba02d695e7a0bd789b40.jpg",
      }
    }
    style={
      Object {
        "height": 400,
        "marginVertical": 20,
        "width": 400,
      }
    }
  />
</View>
`;
